backend/mitbattlecode-496f64277523.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
```

Comment:
noting that we should rebase, to drop this entirely from commit history

Reply:
yeah my bad one this one i didn't realize it was in my staged changes

Reply:
Is good that this key was already revoked a while ago.